src/postgraphql/withPostGraphQLContext.ts

```diff
@@ -135,6 +140,14 @@ async function setupPgClientTransaction ({
   // Sql query.
   const localSettings = new Map<string, mixed>()
 
+  // Set the custom provided settings before jwt claims and role are set
+  // this prevents an accidentional overwriting
+  if (typeof pgSettings === 'object') {
+    for (const key of Object.keys(pgSettings)) {
+      localSettings.set(key, pgSettings[key])
```

Comment:
I think the user should handle that themself with JSON.serialize. I just want to ensure that the types are clear so that `set_config(text, text, bool)` isn't mistaken for a potential different user-defined function such as `set_config(test, json, bool)` etc.